marketplace/deployer_util/wait_for_ready.py

```diff
@@ -161,14 +185,33 @@ def is_ingress_ready(resource):
   return False
 
 
+def is_deployer_job(resource):
+  if 'app.kubernetes.io/component' in labels(resource):
+    return (labels(resource)['app.kubernetes.io/component'] ==
```

Comment:
The UI doesn't currently populate this annotation. I will follow up with a CL to update it.